worlds/rabi_ribi/logic_helpers.py

```diff
@@ -380,25 +375,25 @@ def convert_existing_rando_rule_to_ap_rule(existing_rule: object, player: int, r
             "Bunny Amulet": lambda state: can_bunny_amulet(state, player),
             "Bunny Amulet Lv2": lambda state: can_bunny_amulet_2(state, player),
             "Bunny Amulet Lv3": lambda state: can_bunny_amulet_3(state, player),
-            "Piko Hammer Leveled": lambda state: state.has("Piko Hammer", player),
-            "Carrot Bomb Entry": lambda state: state.has("Carrot Bomb", player),
-            "Carrot Shooter Entry": lambda state: state.has("Carrot Shooter", player),
-            "Tm Cocoa": lambda state: state.has("Cocoa Recruit", player),
-            "Tm Ashuri": lambda state: state.has("Ashuri Recruit", player),
-            "Tm Rita": lambda state: state.has("Rita Recruit", player),
-            "Tm Cicini": lambda state: state.has("Cicini Recruit", player),
-            "Tm Saya": lambda state: state.has("Saya Recruit", player),
-            "Tm Syaro": lambda state: state.has("Syaro Recruit", player),
-            "Tm Pandora": lambda state: state.has("Pandora Recruit", player),
-            "Tm Nieve": lambda state: state.has("Nieve Recruit", player),
-            "Tm Nixie": lambda state: state.has("Nixie Recruit", player),
-            "Tm Aruraune": lambda state: state.has("Aruraune Recruit", player),
-            "Tm Seana": lambda state: state.has("Seana Recruit", player),
-            "Tm Lilith": lambda state: state.has("Lilith Recruit", player),
-            "Tm Vanilla": lambda state: state.has("Vanilla Recruit", player),
-            "Tm Chocolate": lambda state: state.has("Chocolate Recruit", player),
-            "Tm Kotri": lambda state: state.has("Kotri Recruit", player),
-            "Tm Keke Bunny": lambda state: state.has("Keke Bunny Recruit", player),
+            "Piko Hammer Leveled": lambda state: state.has(ItemName.piko_hammer, player),
```

Comment:
Side note, if you ever update the logic, you might want to take a look at OpBacktrack (mentioned in the documentation here)

Reply:
Yeah, I noticed the commented out prereqs when looking at the constraints, but I haven't thought too much about it yet.